warehouse/models/mart/gtfs/fct_monthly_shapes.sql

```diff
@@ -0,0 +1,70 @@
+{{ config(materialized='table') }}
+
+WITH dim_gtfs_datasets AS (
+    SELECT *
+    FROM {{ ref('dim_gtfs_datasets') }}
+),
+
+fct_scheduled_trips AS (
+    SELECT *
+    FROM {{ ref('fct_scheduled_trips') }}
+    WHERE service_date >= '2023-07-29' AND service_date <= '2023-07-31'
+),
+
+dim_shapes_arrays AS (
+
+    SELECT
+        key,
+        base64_url,
+        shape_id,
+        pt_array
+
+    FROM {{ ref('dim_shapes_arrays') }}
+    WHERE {{ incremental_where(default_start_var='GTFS_SCHEDULE_START',
+                               this_dt_column='_feed_valid_from',
```

Comment:
`this_dt_column` needs to refer to a column that is present on the final table being defined here... Could potentially use `month_last_day`